libs/libtins/Makefile

```diff
@@ -0,0 +1,102 @@
+##############################################
+#
+# Copyright (C) 2017 OpenWrt.org
```

Comment:
Regarding the comment abount including libraries if there is no actual application using it: When this PR is accepted, I'll create an additional PR for noddos (https://github.com/noddos/noddos), which has a dependency on libtins